packages/apps/job-launcher/server/src/modules/payment/payment.service.ts

```diff
@@ -70,10 +70,8 @@ export class PaymentService {
   ): Promise<string> {
     const { amount, currency } = dto;
 
-    const amountInCents = Math.round(amount * 100);
-    
     const params: Stripe.PaymentIntentCreateParams = {
-      amount: amountInCents,
+      amount: mul(amount, 100),
```

Comment:
it's not working.
Here will be Invalid integer error, I have described this problems already. It happens because Stripe’s API usually requires integer values for amounts and quantities. For example, an amount of $56.99 would be represented as 5699 in cents.

Reply:
Can you show an example where it does not work?
Request 
{
  "amount": 56.99,
  "currency": "usd"
}
Stripe
![image](https://github.com/humanprotocol/human-protocol/assets/61605646/32e29bb1-09f5-4bc2-b881-1fc22cf259c7)
Database
![image](https://github.com/humanprotocol/human-protocol/assets/61605646/9ffceedd-c747-4907-bfa6-b5150365510c)

Reply:
I will share

Reply:
@portuu3 
```{
  "amount": 0.600002022002020,
  "currency": "eur"
}
{
  "statusCode": 400,
  "message": "Invalid integer: 60.000202200202"
}

{
  "amount": 56.0099,
  "currency": "usd"
}
{
  "statusCode": 400,
  "message": "Invalid integer: 5600.99"
}

{
  "amount": 4.199,
  "currency": "usd"
}
{
  "statusCode": 400,
  "message": "Invalid integer: 419.9"
}```

Reply:
This error occurs when passing to the stripe the amount of the containing precision more than 2 decimal places.